### PR TITLE
add librewolf, bitwig icons and mimetypes by Yoiannides

### PR DESCRIPTION
**Pull request by iaacornus**
associate more icons and mimetypes with bitwig files (#2) along with librewolf icons:
- [X] bitwig `.bwpreset`, `.bwproject`, `.bwclip`, `.multisample`
- [ ] separate icons for .bwtemplate
- [ ] .bwpreset
- [ ] .bwproject
- [x] meson maintenance
- [x] add mimetypes for bwproject folders and other mimetypes for bitwig studio

### CODE REVIEW

File: mimes/scalable/meson.build [hunk 1]
```diff
@@ -448,6 +449,15 @@ link_files = {
         'application-x-java.svg',
         'text-x-java-source.svg',
     ],
+    'application-x-bitwig-studio.svg': [
```
Comment (iaacornus) [inline mimes/scalable/meson.build:452]: also add project, project-folders and template and preset

**Review by iaacornus — COMMENTED** [review]
associate more bitwig-related files

**Comment by iaacornus** [comment]
@yioannides , what do you think of these icons for `.bwtemplates`, `.bwproject` and `.bwpreset`, respectively:

image
image
image

**Comment by yioannides** [comment]
I like the multiple colors, but the drop shadows don't follow GNOME's HIG, so I would go with my MIME icon, which I modeled after other Morewaita MIME icons that follow GNOME's guidelines.

(however, if you have plans on taking this project to a new design direction away from Morewaita and GNOME HIG, (like [Skeuowaita](https://github.com/RusticBard/Skeuowaita) did for example), then feel free to import your designs!)

**Comment by iaacornus** [comment]
> I like the multiple colors, but the drop shadows don't follow GNOME's HIG, so I would go with my MIME icon, which I modeled after other Morewaita MIME icons that follow GNOME's guidelines.
> 
> (however, if you have plans on taking this project to a new design direction away from Morewaita and GNOME HIG, (like [Skeuowaita](https://github.com/RusticBard/Skeuowaita) did for example), then feel free to import your designs!)

yes, that's a mockup taken from papirus. i'll remove the shadows and make it flat to conform with gnome's hig. Im thinking of associating your mime with `.bwpreset` as you intended and use the black for `.bwproject` and the white for `.bwtemplate`. I'll further revamp the icon based on papirus icon theme to make it fit with adwaita

**Comment by yioannides** [comment]
Seeing that Somepaulo is almost done implementing the Bitwig MIME icons, it might be a good idea to start a whole new project called `neowaita` like you suggested and follow your own hybrid style (which I really liked, honestly) or perhaps ask Somepaulo to grant you collaborator access to maintain Morewaita when he's not available. I think in its current state, there might be too large of a cross-development overlap for this repo to continue as a Morewaita fork

**Pull request merged**